README.md

```diff
@@ -45,6 +46,8 @@ server:
 - **log**: Display request info on the console. Always enabled in debug mode.
 - **ip**: Server IP
 - **compress**: Enable GZIP compression
+- **cache**: Enable cache for rendered content
+  - Enable the cache will let hexo-server respond to the second request much quicker, but your changes will no longer make effects in real time.
```

Comment:
```
  - This can speed up server response.
  - However, any changes will no longer take effect in real time. Suitable for production environment only.
```

Reply:
Updated.